cloud/azure/variables.tf

```diff
@@ -0,0 +1,25 @@
+variable "application_name" {
+  type        = string
+  description = "Azure Web App Name"
+}
+
+variable "docker_username" {
+  type        = string
+  description = "Docker username"
+}
+
+variable "docker_repository_name" {
+  type        = string
+  description = "Name of container image"
+}
+
+variable "docker_password" {
+  type        = string
+  description = "Password to log in to docker"
+}
+
+variable "tag_name" {
```

Comment:
image_tag_name maybe?  I think tag will apply to too many concepts now and later.

Reply:
renamed in last commit